generators/app/templates/express-api/Dockerfile

```diff
@@ -1,11 +1,12 @@
-FROM alpine:3.3
```

Comment:
@beverts312 is this better than the official node alpine image? https://hub.docker.com/_/node/
FROM node:7-apline